config/openstack/group_vars/k8s_cluster/ck8s-k8s-cluster-openstack.yaml

```diff
@@ -12,6 +12,7 @@ cinder_csi_enabled: true
 persistent_volumes_enabled: true
```

Comment:
Any reason to only have this for openstack?
I'm guessing we could run this on exoscale for instance as well.

Reply:
That is true !! 

do you recommend moving it to ck8s-k8s-cluster.yaml ?

Reply:
Yea, I think that makes sense. It should work for any as long as you set it up correctly. But you've specified in the docs that it's only tested on openstack so that should be fine for now imo.

Reply:
But when we do an init with non openstack clouds, these new values won't show up in the rendered manifests. I also want to propose moving it to  ck8s-k8s-cluster.yaml

Reply:
This does not seem to work for Cleura
We need to test and make it work on all cloud providers

Reply:
> This does not seem to work for Cleura We need to test and make it work on all cloud providers

No we don't. This is a public repo so I will skip the internal business related stuff but local storage is to only be offered, initially, on the infrastructure providers elastx and safespring.

Reply:
> But when we do an init with non openstack clouds, these new values won't show up in the rendered manifests, so I want to propose moving the values to ck8s-k8s-cluster.yaml

Any thoughts ? if not, i'll keep them in openstack manifests since we are currently only offering it in elastx and safespring.

Reply:
> But when we do an init with non openstack clouds, these new values won't show up in the rendered manifests, so I want to propose moving the values to ck8s-k8s-cluster.yaml

So, if you move the values to `ck8s-k8s-cluster.yaml` they will show up when doing an init on non-openstack clouds?